packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -88,6 +94,14 @@ function onDidEndTaskProcessHandler(event: vscode.TaskProcessEndEvent): void {
   if (task.scope !== undefined && isTeamsfxTask(task)) {
     allRunningTeamsfxTasks.delete({ source: task.source, name: task.name, scope: task.scope });
   } else if (isNpmInstallTask(task)) {
+    try {
+      ExtTelemetry.sendTelemetryEvent(TelemetryEvent.DebugNpmInstallStop, {
+        [TelemetryProperty.DebugNpmInstallExitCode]: event.exitCode + "",
```

Comment:
is it possible to add the output message from terminal to this telemetry event if the npm-install task is failed?

Reply:
It seems not possible because `event` only has properties `task` and `terminate()`. And `task` property is the task description, and it does not contain any runtime info.